### ISSUE TITLE
nanobind support?

### ISSUE THREAD

**Issue by wojdyr** [mosra/m.css#261]
I've been using m.css for a pybind11 module for a few years.
Recently, I migrated this module to [nanobind](https://github.com/wjakob/nanobind/), which is a next-gen library from the same author as pybind11.
I wonder, are there any plans for nanobind support in m.css?
If not, any hints what could be used as a replacement?

**Comment by wojdyr** [comment]
Great, thanks! I don't have any time frame and can easily wait a couple months.

Nanobind can natively generate pyi stub files. I was thinking, could this be used as an input for m.css?

**Comment by mosra** [comment]
Not really. It'd be one extra indirection, meaning an extra chance for bugs to appear, and as the stubs usually remove all traces of the underlying support library, it'd be no longer possible to detect that nanobind was used and apply nanobind-specific workarounds for it.
